lib/features/announcement/ui/announcements_list/navigation_drawer.dart

```diff
@@ -0,0 +1,148 @@
+                  ),
+                  const SizedBox(height: 6,),
+                  NavigationItem(
+                      name: const LocalizedButtonResolver().chats(context),
+                      leadingIcon: Icons.chat,
+                      onPressedFunction: ()=> itemPressed(context, index:0)
+                  ),
+                  const SizedBox(height: 6,),
+                  NavigationItem(
+                      name: const LocalizedButtonResolver().calendar(context),
+                      leadingIcon: Icons.calendar_month,
+                      onPressedFunction: ()=> itemPressed(context, index:0)
+                  ),
+                  const SizedBox(height: 6,),
+                  NavigationItem(
+                      name: const LocalizedButtonResolver().discussionForum(context),
+                      leadingIcon: Icons.post_add,
+                      onPressedFunction: ()=> itemPressed(context, index:0)
+                  ),
+                  const SizedBox(height: 6,),
+                  NavigationItem(
+                      name: const LocalizedButtonResolver().support(context),
+                      leadingIcon: Icons.notification_add_rounded,
+                      onPressedFunction: ()=> itemPressed(context, index:0)
+                  ),
+                  const SizedBox(height: 6,),
+                  Divider(
+                    height: 20,
+                    thickness: 2,
+                    indent: 20,
+                    endIndent: 10,
+                    color: Color(( constants.primaryColorDark.toInt() % 0xFF000000 + 0x66000000)),
+                  ),
+                  NavigationItem(
+                      name: const LocalizedButtonResolver().settings(context),
+                      leadingIcon: Icons.settings,
+                      onPressedFunction: ()=> itemPressed(context, index:0)
+                  ),
+                  const SizedBox(height: 6,),
+                  NavigationItem(
+                      name: const LocalizedButtonResolver().notifications(context),
+                      leadingIcon: Icons.notification_add_rounded,
+                      onPressedFunction: ()=> itemPressed(context, index:0)
+                  ),
+                  const SizedBox(height: 6,),
+                  NavigationItem(
+                      name: const LocalizedButtonResolver().helps(context),
+                      leadingIcon: Icons.settings,
+                      onPressedFunction: ()=> itemPressed(context, index:0)
+                  ),
+                  const SizedBox(height: 6,),
+                ],
+            ),
+          ),
+        ),
+      )
+    );
+  }
+}
+
+itemPressed(BuildContext context, {required int index}) {
+  Navigator.pop(context);
+  switch(index){
+    case 0:
+      Navigator.push(context, MaterialPageRoute(
+          builder: (context) =>
+          const VocelSetting(),
+          settings: const RouteSettings(arguments: "settings page")));
```

Comment:
Should this string be internationalized?>

Reply:
Yep, I will change all implementations for different routes, and I have not figured it out. These will be changed